specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/stable/2022-02-01/VirtualNetworkGatewayConnections.json

```diff
@@ -0,0 +1,275 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-02-01",
+    "title": "NetworkAdminManagementClient",
+    "description": "The Admin Network Management Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network.Admin/adminVirtualNetworkGatewayConnections": {
+      "get": {
```

Comment:
In addition to the list API, please define & implement below API for GET on individual resource. 

GET /subscriptions/{subscriptionId}/providers/Microsoft.Network.Admin/adminVirtualNetworkGatewayConnections/{connectionName}